cmd/admin-revoker/main.go

```diff
@@ -235,6 +229,24 @@ func main() {
 				}
 			},
 		},
+		{
+			Name:  "auth-revoke",
+			Usage: "Revoke all pending/valid authorizations for a domain",
+			Action: func(c *cli.Context) {
+				domain := c.Args().First()
+				_, _, _, sac, stats := setupContext(c)
+				ident := core.AcmeIdentifier{Value: domain, Type: core.IdentifierDNS}
+				authsRevoked, pendingAuthsRevoked, err := sac.RevokeAuthorizationsByDomain(ident)
+				cmd.FailOnError(err, fmt.Sprintf("Failed to revoke authorizations for %s", ident.Value))
+				fmt.Printf(
```

Comment:
In `revokeBySerial`, we use auditLogger.Info, which I think is appropriate here.